easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-5.4.0-CrayIntel-2015.11.eb

```diff
@@ -0,0 +1,46 @@
+# @author: gppezzi, lucamar (CSCS)
+easyblock = "ConfigureMake"
```

Comment:
well, one reason to enhance the existing QuantumESPRESSO easyblock is the extensive sanity check that is being done by it... Also, EasyBuild patches the `make.sys` file generated by the `configure` script to ensure that a bunch of things are set correctly (w.r.t. BLAS/LAPACK, for example).

It would be nice to get the existing QuantumESPRESSO easyblock updated to make it cover newer versions too...

It's possible a couple of enhancements are needed to make it work with Cray too, although it looks quite OK at first sight.